examples/openthread/README.md

```diff
@@ -0,0 +1,46 @@
+## OpenThread on RIOT
+
+This test demonstrates how to use the [OpenThread](https://github.com/openthread/openthread)
+open source implementation of [Thread](https://threadgroup.org/) on RIOT.
+ The [Command Line Interface](https://github.com/openthread/openthread/blob/master/examples/apps/cli/README.md) of
+OpenThread was ported. Please check the [full documentation]
+(https://github.com/openthread/openthread/blob/master/src/cli/README.md) of the CLI for usage information.
+
+You can either build a FTD or MTD firmware:
+- MTD: A Minimal Thread Device does not have router functionality compiled in. An MTD may or may not be sleepy.
+- FTD: A Full Thread Device has router functionality compiled in.
+
+## Quick usage
+
+With RIOT port, a node is auto-setup and ready to communicate with this configuration:
+OPENTHREAD_PANID: 0xbeef
```

Comment:
Maybe surround the configuration with triple back ticks. I would also write them using the makefile variable assignment syntax, e.g with `=`